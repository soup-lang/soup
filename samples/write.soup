write: console.log
write(0)
write(1)
write(-1)
write(3.14159)
write(2.71828)
write(true)
write(false = true)
write([])
write([0, 1])
write([ [3.14159], [2.71828] ])
write([false, true])
