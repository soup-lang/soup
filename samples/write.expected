0
1
-1
3.14159
2.71828
true
false
[]
[ 0, 1 ]
[ [ 3.14159 ], [ 2.71828 ] ]
[ false, true ]
